projects/cdk/schematics/ng-update/v4/steps/constants/identifiers-to-replace.ts

```diff
@@ -1244,4 +1244,70 @@ export const IDENTIFIERS_TO_REPLACE: ReplacementIdentifierMulti[] = [
             moduleSpecifier: '@taiga-ui/addon-mobile',
         },
     },
+    {
+        from: {
+            name: 'TuiMobileDialogModule',
+            moduleSpecifier: '@taiga-ui/addon-commerce',
+        },
+        to: {
+            name: 'TuiMobileDialogComponent',
+            moduleSpecifier: '@taiga-ui/addon-mobile',
+        },
+    },
+    {
+        from: {
+            name: 'TuiRippleModule',
+            moduleSpecifier: '@taiga-ui/addon-commerce',
+        },
+        to: {
+            name: 'TuiRippleDirective',
+            moduleSpecifier: '@taiga-ui/addon-mobile',
+        },
+    },
+    {
+        from: {
+            name: 'TuiTouchableModule',
+            moduleSpecifier: '@taiga-ui/addon-commerce',
+        },
+        to: {
+            name: 'TuiTouchableDirective',
+            moduleSpecifier: '@taiga-ui/addon-mobile',
+        },
+    },
+    {
+        from: {
+            name: 'TuiElasticStickyModule',
+            moduleSpecifier: '@taiga-ui/addon-commerce',
```

Comment:
from `addon-mobile` ? (and below and above)

Reply:
fixed